07_control_statements_branching_and_jumps/exercises/count_ei.c

```diff
@@ -0,0 +1,25 @@
+// Write a program that reads input up to # and reports the number of times 
+// that the sequence ei occurs.Note: The program will have to “remember” the 
+// preceding character as well as the current character. Test it with input 
+// such as Receive your eieio award.
+#include <stdio.h>
+
+int main(void)
+{
+    char ch;
+    int count = 0;
+
+    while ((ch = getchar()) != '#')
+    {
+        if (ch == 'e')
+        {    
+            ch = getchar(); 
+            if (ch == 'i')
```

Comment:
This is a bug, because _#_ may come after _e_.  

Hint: In each iteration of the loop don't check what goes after _e_, check what was before _i_.